src/main/webapp/script.js

```diff
@@ -0,0 +1,31 @@
+function loadBody() {
+  console.log("Verifying login...");
+  verifyLogin();
+}
+
+/** Contacts Login servlet to verify if user is logged in. */
+function verifyLogin() {
+  console.log("Fetching login status...");
+  fetch('/login').then(response => response.json()).then((arrStrings) => {
+    const loginElement = document.getElementById("login-box");
+    if (arrStrings[0] == "false") {
+      document.getElementById("school-name").style.display = "none";
```

Comment:
This is probably unnecessary if you're just going to redirect the user to a login page